sdk/framework/backend.go

```diff
@@ -486,9 +486,18 @@ func (b *Backend) handleRootHelp() (*logical.Response, error) {
 		return nil, err
 	}
 
+	// The plugin type (e.g. "kv", "cubbyhole") is only assigned at the time
+	// the plugin is enabled (mounted). If specified in the request, the type
+	// will be used as part of the request/response body names in the OAS
+	// document
+	var requestResponsePrefix string
```

Comment:
```suggestion
	requestResponsePrefix := req.GetString("requestResponsePrefix")
```
... and you can remove the assignment below.

Reply:
TIL, thanks!